homeassistant/components/dynalite/config_flow.py

```diff
@@ -19,14 +20,16 @@ def __init__(self) -> None:
         """Initialize the Dynalite flow."""
         self.host = None
 
-    async def async_step_import(self, import_info: dict[str, Any]) -> Any:
+    async def async_step_import(self, import_info: MappingProxyType[str, Any]) -> Any:
```

Comment:
I'm really confused about what is happening in this PR. This method is not typed correctly, as a step receives a dictionary, not a MappingProxyType.